### DISCUSSION TITLE
Another Category

### DISCUSSION THREAD

**Discussion by duarch** [cssbattle/forum#15] [Ideas]
Maybe we should have another ranking category like: **less properties, less selectors, less elements**. 

I know it sounds like redundant. But in this category, the amount of characters would not be the 2nd main scoring criteria. 
The idea is to create a category that could help us develop abilities to use CSS in the real life production environment. 

I also know that the main ideia of "code golf" is to achieve the shortest code possible. And it's awesome, I love it! 
The another category would be more educational than competitive itself. By the way it would be nice if we could choose show our code or not.

**Comment by gunnarbittersmann** [comment]
Hm, wouldn’t that be boring? Every target can be solved with one selector and one property:`*+*{background:…}`.

**Reply by gunnarbittersmann** [reply]
When you consider code length as well, it makes sense.
That’s another battle: solve all targets with only `*+*{background:…}` in as few characters as possible.
Just for fun, I did this for [#11](https://cssbattle.dev/play/11); my current best is 266 characters.

**Comment by kushsolitary** [comment]
@duarch we already have something planned along those lines https://cssbattle.dev/learn :)
